### PR TITLE
Add component jest tests

### PR DESCRIPTION
**Pull request by kurtis-casperson**
#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS Tablet
- [ ] Tested on small Android
- [ ] n/a

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1201939368270006

### CODE REVIEW

File: src/__tests__/components/ClickableRow.test.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    const props = {
+      onPress: () => undefined | (async () => undefined),
+      onLongPress: () => undefined | (async () => undefined),
+      highlight: 11,
+      gradient: 11,
+      autoHeight: 11,
+      children: 'String',
+      underline: 11,
+      // eslint-disable-next-line react/no-unused-prop-types
+      marginRem: [11] | 11,
```
Comment (thehobbit85) [inline src/__tests__/components/ClickableRow.test.js:23]: This should not have the 'or' sign, it was only part of the type.
You should pass just the number 11.

File: src/__tests__/components/ClickableRow.test.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      // eslint-disable-next-line react/no-unused-prop-types
+      paddingRem: [11] | 11,
```
Comment (thehobbit85) [inline src/__tests__/components/ClickableRow.test.js:25]: This should not have the 'or' sign, it was only part of the type.
You should pass just the number 11.

File: src/__tests__/components/DataRow.test.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import { DataRowComponent } from '../../components/themed/DataRow.js'
+
+describe('DataRow', () => {
+  it('should render with loading props', () => {
+    const renderer = new ShallowRenderer()
+    const props = {
+      label: 'string',
+      value: 11,
+      // eslint-disable-next-line react/no-unused-prop-types
+      marginRem: [11] | 11,
```
Comment (thehobbit85) [inline src/__tests__/components/DataRow.test.js:17]: This should not have the 'or' sign, it was only part of the type.
You should pass just the number 11.

File: src/__tests__/components/EdgeTextField.test.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+import { getTheme } from '../../components/services/ThemeContext.js'
+import { EdgeTextFieldComponent as Request } from '../../components/themed/EdgeTextField.js'
+
+describe('Request', () => {
+  it('should render with loading props', () => {
+    const renderer = new ShallowRenderer()
+
+    const props = {
+      marginRem: 11 | [11],
```
Comment (thehobbit85) [inline src/__tests__/components/EdgeTextField.test.js:15]: This should not have the 'or' sign, it was only part of the type.
You should pass just the number 11.

File: src/__tests__/components/OutlinedTextInput.test.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+describe('OutlinedTextInput', () => {
+  it('should render with some props', () => {
+    const renderer = new ShallowRenderer()
+
+    const props = {
+      value: 'string',
+      error: 'string',
+      label: 'string',
+      clearIcon: true, // Defaults to 'true'
+      marginRem: 0.5 | [0.5], // Defaults to 0.5
```
Comment (thehobbit85) [inline src/__tests__/components/OutlinedTextInput.test.js:18]: This should not have the 'or' sign, it was only part of the type.
You should pass just the number 0.5.

File: src/__tests__/components/OutlinedTextInput.test.js [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+      multiline: true, // Defaults to 'false'
+      searchIcon: true, // Defaults to 'false'
+
+      onBlur: () => undefined,
+      onChangeText: () => undefined,
+      onClear: () => undefined,
+      onFocus: () => undefined,
+
+      // Other React Native TextInput properties:
+      autoCapitalize: 'none' | 'sentences' | 'words' | 'characters', // Defaults to 'sentences'
```
Comment (thehobbit85) [inline src/__tests__/components/OutlinedTextInput.test.js:28]: This should not have the 'or' sign, it was only part of the type.
You should just pass one of the options.

File: src/__tests__/components/OutlinedTextInput.test.js [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+      autoCorrect: true, // Defaults to 'true'
+      blurOnSubmit: true, // Defaults to 'true'
+      inputAccessoryViewID: 'string',
+      keyboardType: 'default' | 'number-pad' | 'decimal-pad' | 'numeric' | 'email-address' | 'phone-pad', // Defaults to 'default'
```
Comment (thehobbit85) [inline src/__tests__/components/OutlinedTextInput.test.js:32]: This should not have the 'or' sign, it was only part of the type.
You should just pass one of the options.

File: src/__tests__/components/OutlinedTextInput.test.js [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+      maxLength: 11,
+      onSubmitEditing: () => undefined,
+      returnKeyType: 'done' | 'go' | 'next' | 'search' | 'send', // Defaults to 'done'
```
Comment (thehobbit85) [inline src/__tests__/components/OutlinedTextInput.test.js:35]: This should not have the 'or' sign, it was only part of the type.
You should just pass one of the options.

File: src/__tests__/components/ExchangeQuoteComponent.test.js [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+
+import { getTheme } from '../../components/services/ThemeContext.js'
+import { ExchangeQuoteComponent } from '../../components/themed/ExchangeQuoteComponent.js'
+
+describe('ExchangeQuote', () => {
+  it('should render with loading props', () => {
+    const renderer = new ShallowRenderer()
+
+    const props = {
+      isTop: true | undefined,
```
Comment (thehobbit85) [inline src/__tests__/components/ExchangeQuoteComponent.test.js:15]: This should not have the 'or' sign, it was only part of the type.
You should just pass one of the options.

File: src/__tests__/components/ExchangeQuoteComponent.test.js [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+      walletIcon: 'string',
+      cryptoAmount: 'string',
+      currency: 'string',
+      currencyCode: 'string',
+      fiatCurrencyCode: 'string',
+      fiatCurrencyAmount: 'string',
+      walletName: 'string',
+      total: 'string',
+      miningFee: 'string' | undefined,
```
Comment (thehobbit85) [inline src/__tests__/components/ExchangeQuoteComponent.test.js:24]: This should not have the 'or' sign, it was only part of the type.
You should just pass one of the options.

File: src/__tests__/components/FlipInput.test.js [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+        nativeAmount: 'string'
+      },
+      isEditable: true,
+      isFiatOnTop: true,
+      isFocus: true,
+
+      topReturnKeyType: 'string',
+      inputAccessoryViewID: 'string',
+      headerText: 'string',
+      headerLogo: 'string' | undefined,
```
Comment (thehobbit85) [inline src/__tests__/components/FlipInput.test.js:41]: This should not have the 'or' sign, it was only part of the type.
You should just pass one of the options.

File: src/__tests__/components/HiddenMenuButtons.test.js [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+      },
+      left: {
+        label: 'string',
+        children: 'string',
+        color: 'success' | 'danger' | 'default',
+        onPress: async () => undefined || undefined
+      },
+      isSwipingRight: true,
+      isSwipingLeft: true,
+      swipeDirection: 'left' | 'right' | undefined,
```
Comment (thehobbit85) [inline src/__tests__/components/HiddenMenuButtons.test.js:41]: This should not have the 'or' sign, it was only part of the type.
You should just pass one of the options.

File: src/__tests__/components/HiddenMenuButtons.test.js [hunk 13]
```diff
@@ -0,0 +1,48 @@
...
+        label: 'string',
+        children: 'string',
+        color: 'success' | 'danger' | 'default',
+        onPress: async () => undefined || undefined
+      },
+      left: {
+        label: 'string',
+        children: 'string',
+        color: 'success' | 'danger' | 'default',
+        onPress: async () => undefined || undefined
```
Comment (thehobbit85) [inline src/__tests__/components/HiddenMenuButtons.test.js:37]: This is redundant, just return one 'undefined'

File: src/__tests__/components/HiddenMenuButtons.test.js [hunk 14]
```diff
@@ -0,0 +1,48 @@
...
+      },
+      leftSwipable: {
+        label: 'string',
+        children: 'string',
+        color: 'success' | 'danger' | 'default',
+        onPress: async () => undefined || undefined
```
Comment (thehobbit85) [inline src/__tests__/components/HiddenMenuButtons.test.js:25]: This is redundant, just return one 'undefined'

File: src/__tests__/components/HiddenMenuButtons.test.js [hunk 15]
```diff
@@ -0,0 +1,48 @@
...
+describe('Request', () => {
+  it('should render with loading props', () => {
+    const renderer = new ShallowRenderer()
+
+    const props = {
+      rightSwipable: {
+        label: 'string',
+        children: 'string',
+        color: 'success' | 'danger' | 'default',
+        onPress: async () => undefined || undefined
```
Comment (thehobbit85) [inline src/__tests__/components/HiddenMenuButtons.test.js:19]: This is redundant, just return one 'undefined'

File: src/__tests__/components/TransactionListRow.test.js [hunk 16]
```diff
@@ -0,0 +1,35 @@
...
+      isSentTransaction: true,
+      requiredConfirmations: 11,
+      selectedCurrencyName: 'string',
+      thumbnailPath: 'string',
+      walletBlockHeight: 11,
+      // eslint-disable-next-line react/no-unused-prop-types
+      walletId: 'string',
+      // eslint-disable-next-line react/no-unused-prop-types
+      currencyCode: 'string',
+      transaction: 'string' | 11,
```
Comment (thehobbit85) [inline src/__tests__/components/TransactionListRow.test.js:28]: This should not have the 'or' sign, it was only part of the type.
You should just pass one of the options.

File: src/__tests__/components/WalletListCurrencyRow.test.js [hunk 17]
```diff
@@ -0,0 +1,39 @@
...
+  it('should render with loading props', () => {
+    const renderer = new ShallowRenderer()
+
+    const props = {
+      currencyCode: 'string',
+      gradient: true,
+      onPress: () => undefined,
+      onLongPress: () => undefined,
+      showRate: true,
+      paddingRem: 11 | [11],
```
Comment (thehobbit85) [inline src/__tests__/components/WalletListCurrencyRow.test.js:20]: This should not have the 'or' sign, it was only part of the type.
You should just pass one of the options.

File: src/__tests__/passwordReminderReducer.test.js [hunk 18]
```diff
@@ -1,270 +0,0 @@
-// @flow
```
Comment (thehobbit85) [inline src/__tests__/passwordReminderReducer.test.js:1]: Why did you remove this file?
Isn't it a working test?

**Review by thehobbit85 — CHANGES_REQUESTED** [review]
Go over it again, and make sure you don't pass any variables with the '|' symbol.

File: src/__tests__/passwordReminderReducer.test.js (hunk 18 above)
Comment (kurtis-casperson) [reply src/__tests__/passwordReminderReducer.test.js:1]: I didn't think reducer files were supposed to be in the tests?

Comment (kurtis-casperson) [reply src/__tests__/passwordReminderReducer.test.js:1]: PasswordReminderModal.js 
is the only password reminder file in themed components

File: src/__tests__/components/TransactionRow.test.js [hunk 19]
```diff
@@ -60,7 +60,7 @@ describe('Transaction List Row', () => {
       symbol: '‎ɱ'
     }
     // CryptoAmount
-    const cryptoAmount = convertNativeToDisplay(displayDenomination.multiplier)(abs(transaction.nativeAmount || ''))
```
Comment (thehobbit85) [inline src/__tests__/components/TransactionRow.test.js:63]: Don't remove '||', use '??' instead

**Review by thehobbit85 — CHANGES_REQUESTED** [review]

File: src/__tests__/components/ClickableRow.test.js [hunk 20]
```diff
@@ -0,0 +1,32 @@
...
+
+import { getTheme } from '../../components/services/ThemeContext.js'
+import { ClickableRowComponent as Request } from '../../components/themed/ClickableRow.js'
+
+describe('Request', () => {
+  it('should render with loading props', () => {
+    const renderer = new ShallowRenderer()
+
+    const props = {
+      onPress: () => undefined | (async () => undefined),
```
Comment (thehobbit85) [inline src/__tests__/components/ClickableRow.test.js:15]: Another '|' that needs to be removed.

File: src/__tests__/components/ClickableRow.test.js [hunk 21]
```diff
@@ -0,0 +1,32 @@
...
+      onLongPress: () => undefined | (async () => undefined),
```
Comment (thehobbit85) [inline src/__tests__/components/ClickableRow.test.js:16]: Another '|' that needs to be removed.

**Review by thehobbit85 — CHANGES_REQUESTED** [review]
